src/main/java/org/apache/gossip/crdt/GrowOnlyCounter.java

```diff
@@ -0,0 +1,94 @@
+package org.apache.gossip.crdt;
+
+
+import java.lang.annotation.ElementType;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GrowOnlyCounter implements CrdtCounter<Long, GrowOnlyCounter> {
+
+
+    private final Map<String, Long> counters = new HashMap<>();
+    private final String myID;
```

Comment:
Comment above about myId it should be myId not myID and I think we wish it to be transient or not serialized as it crosses the wire.